### DISCUSSION TITLE
2 Wallboxe PV Überschuss

### DISCUSSION THREAD

**Discussion by Trunks1982** [camueller/SmartApplianceEnabler#508] [Allgemein]
Kann das sein das SMA es immer noch nicht kann 2 Wallboxen PV Überschussladen einzustellen?
Kann ja eigentlich nicht so schwer sein sich den Überschuss zuteilen.

**Comment by camueller** [comment]
Kannst Du das Problem genauer beschreiben und mir ein Log-File vom SAE bereitstellen, mit dem sich das nachvollziehen lässt? Falls es so ist, wie Du sagst, würde ich auf SMA zugehen und das klären.
